lld/test/ELF/linkerscript/nocrossrefs.test

```diff
@@ -0,0 +1,179 @@
+# REQUIRES: x86
+# RUN: rm -rf %t && split-file %s %t && cd %t
+
+# RUN: llvm-mc --triple=x86_64-unknown-linux -filetype=obj -o main.o main.s
+# RUN: not ld.lld main.o -o main --script script1.ld 2>&1 | FileCheck -check-prefix=ERR %s
```

Comment:
`-o ` is unneeded when using `not ld.lld`
(Since we have done `cd %t`, the unused output file will not clutter the current directory)